templates/invoice.html

```diff
@@ -0,0 +1,356 @@
+            <span class="header-item">{% price order.total_gross %}</span>
+            <br />
+        </div>
+    </div>
+    <div class="section-left">
+        <div class="content-padded">
+            <span class="header-title">
+                {% trans "BILLING ADDRESS" context "Invoice PDF" %}
+            </span>
+            <br />
+            <span class="normal-text">
+                {{ order.billing_address.first_name }}
+                {{ order.billing_address.last_name }}
+            </span>
+            <br />
+            <span class="normal-text">{{ order.billing_address.street_address_1 }}</span>
+            <br />
+            {% if order.billing_address.street_address_2 %}
+            <span class="normal-text">{{ order.billing_address.street_address_2 }}</span>
+            <br />
+            {% endif %}
+            <span class="normal-text">{{ order.billing_address.country }}</span>
+            <br />
+            <span class="normal-text">{{ order.billing_address.city }}</span>
+            <br />
+            <span class="normal-text">{{ order.billing_address.postal_code }}</span>
+            <br />
+            <span class="normal-text">{{ order.billing_address.phone }}</span>
+            <br />
+            <span class="normal-text">{{ order.get_customer_email }}</span>
+            <br />
+        </div>
+    </div>
+    <div class="section-right">
+        <div class="content-padded">
+            <span class="header-title">
+                {% trans "SHIPMENT ADDRESS" context "Invoice PDF" %}
+            </span>
+            <br />
+            <span class="normal-text">
+                {{ order.shipping_address.first_name }}
+                {{ order.shipping_address.last_name }}
+            </span>
+            <br />
+            <span class="normal-text">{{ order.shipping_address.street_address_1 }}</span>
+            <br />
+            {% if order.shipping_address.street_address_2 %}
+            <span class="normal-text">{{ order.shipping_address.street_address_2 }}</span>
+            <br />
+            {% endif %}
+            <span class="normal-text">{{ order.shipping_address.country }}</span>
+            <br />
+            <span class="normal-text">{{ order.shipping_address.city }}</span>
+            <br />
+            <span class="normal-text">{{ order.shipping_address.postal_code }}</span>
+            <br />
+            <span class="normal-text">{{ order.shipping_address.phone }}</span>
+            <br />
+            <span class="normal-text">{{ order.user_email }}</span>
+            <br />
+        </div>
+    </div>
+    <div class="content-tight-padded">
+        <div class="content-separator">&nbsp;</div>
+    </div>
+    <div class="section-left">
+        <div class="content-padded">
+            <span class="header-title">{% trans "PAYMENT METHOD" context "Invoice PDF" %}</span>
+            <br />
+            <span class="normal-text">Credit Card</span>
```

Comment:
Placeholder?

Reply:
Whoops! Yep, wrrr I knew I was about to change something just forgot what it was.